tools/build/scanCode.py

```diff
@@ -1,3 +1,21 @@
+#!/usr/bin/env python
```

Comment:
I believe you have these changes in two distinct PRs. Should disappear when rebasing.